jjsModule_2.js

```diff
@@ -0,0 +1,109 @@
+// Revealing Module Pattern, a Date utility module that includes:
+// a. **setDate**
+// b. **getDate**
+// c. **getDayName**
+// d. **getMonthName**
+// e. **isFuture**
+// f. **isToday**
+//
+//var module = {};
+var exports = module.exports = {};
```

Comment:
Better to just add these to a global block than redefine if you're trying to avoid a linting error here.